frontend/src/lib/router.tsx

```diff
@@ -721,7 +721,6 @@ const defaultRoutes: {
     },
     useClusterURL: false,
     noAuthRequired: true,
-    disabled: !helpers.isElectron(),
```

Comment:
Since this was added in this PR, can you delete it where it was added? Same for the other change. Unless there are complications I haven't noticed.

Reply:
AFAIU this wasn't added in this PR.

Reply:
This was added in #990

Reply:
Ah, ok. Sorry.